core/capabilities/integration_tests/por/fetchtrueusd_test.go

```diff
@@ -147,12 +169,16 @@ func (n ComputeFetcherFactory) NewFetcher(log commonlogger.Logger, emitter custm
 	}
 }
 
-func setupDons(ctx context.Context, t *testing.T, lggr logger.SugaredLogger, workflowURL string, cronSchedule string,
-	triggerFactory framework.TriggerFactory) *data_feeds_cache.DataFeedsCache {
+func setupDons(ctx context.Context, t *testing.T, lggr logger.SugaredLogger, workflowURL string, cronSchedule string, cronSchedule2 string,
```

Comment:
_nit_ this signature is a bit opaque.  consider returning a named interface of hooks that happen at during execution points (or name the return values)